stubs/sublime_plugin.pyi

```diff
@@ -0,0 +1,114 @@
+import sublime
+from typing import Optional, Union, List, Dict, Tuple
+
+
+class CommandInputHandler():
+    def name(self) -> str: ...
+    def next_input(self, args: dict) -> Optional[CommandInputHandler]: ...
+    def placeholder(self) -> str: ...
+    def initial_text(self) -> str: ...
+    def preview(self, arg: dict) -> Union[str, sublime.Html]: ...
```

Comment:
I haven't verified this right now, but from the documentation, the `arg` received should be the *value* of a ListInputHandler's list item, which may be any serializable, meaning ListInputHandler must be a generic class. 

TextInputHandler would then inherit `CommandInputHandler[str]`.

This means that `ListInputItem` needs to be generic as well, but we don't have that at all currently since it's ST4 only.

Reply:
Gotcha. I only tried on TextInputHandler.
What's the plan for Python 3.8 and ST4? Is there a specific branch I should look at?